src/main/java/com/mraof/minestuck/item/weapon/OnHitEffect.java

```diff
@@ -30,13 +36,15 @@
 import net.minecraft.world.entity.decoration.ArmorStand;
 import net.minecraft.world.entity.item.ItemEntity;
 import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.Item;
 import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.item.enchantment.EnchantmentHelper;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.storage.loot.LootContext;
 import net.minecraft.world.level.storage.loot.LootTable;
 import net.minecraft.world.level.storage.loot.parameters.LootContextParamSets;
 import net.minecraft.world.phys.AABB;
+import org.checkerframework.checker.units.qual.C;
```

Comment:
I assume this is unused (and if not it probably shouldnt be used), check if there are any other unused imports